addons/web/static/tests/webclient/actions/embedded_action.test.js

```diff
@@ -104,11 +104,30 @@ class Pony extends models.Model {
 }
 
 class ResUsersSettings extends WebResUsersSettings {
-    /** @param {number[]} id */
-    get_embedded_actions_settings(id) {
+    /**
+     * @param {number} action_id
+     * @param {number} res_id
+     */
+    get_embedded_actions_setting(id, action_id, res_id) {
+        const kwargs = getKwArgs(arguments, "id", "action_id", "res_id", "vals");
```

Comment:
Do you need `vals`? Also, you can use destructuring
```suggestion
        ({ id, action_id, res_id } = getKwArgs(arguments, "id", "action_id", "res_id"));
```